src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -124,39 +152,57 @@ The `init()` you saw in the [_Overview_]({%- link _documentation/development/sdk
 
 - `scope.set_context(key, value)`: Sets the context key to a value, overwriting a potential previous value.  Removing a key is SDK-defined, either with a `remove_context` function or by passing nothing as data. The types are sdk specified.
 
+- `scope.set_level(level)`: Sets the level of all events sent within this scope.
+
+- `scope.set_transaction(transcation_name)`: Sets the name of the current transaction.
+
 - `scope.set_fingerprint(fingerprint[])`: Sets the fingerprint to group specific events together
 
 - `scope.add_event_processor(processor)`: Registers an event processor function.  It takes an event and returns a new event or `None` to drop it.  This is the basis of many integrations.
 
-- `scope.add_error_processor(processor)`: Registers an error processor function.  It takes an event and exception object and returns a new event or `None` to drop it.  This can be used to extract additional information out of an exception object that the SDK cannot extract itself.
+- `scope.add_error_processor(processor)` (optional): Registers an error processor function.  It takes an event and exception object and returns a new event or `None` to drop it.  This can be used to extract additional information out of an exception object that the SDK cannot extract itself.
 
-- `scope.clear()`: resets a scope to default values (prevents inheriting).  This never clears registered event processors.
+- `scope.clear()`: Resets a scope to default values (prevents inheriting).  This never clears registered event processors.
 
 - `scope.apply_to_event(event[, max_breadcrumbs])`: Applies the scope data to the given event object. This also applies the event processors stored in the scope internally.  Some implementations might want to set a max breadcrumbs count here.
 
+## Client
+
+A Client is the part of the SDK that is responsible for event creation. To give an example, the Client should convert an exception to an Sentry event.  The Client should be stateless, it gets the Scope injected and delegates the work of sending the event to the Transport.
+
+- `Client::from_config(config)`: (alternatively normal ctor) This takes typically an object with options + dsn.
+
+- `Client::capture_event(event, scope)`: Captures the the event by merging it with other data with defaults from the client + if a scope is passed to this system the data from the scope and passes it on to the internal transport.
+
+- `Client::close(timeout)`: Flushes out the queue for up to timeout seconds. If the client can guarantee delivery of events only up to the current point in time this is preferred. This might block for timeout seconds. The client should be disabled or disposed after close is called
+
+- `Client::flush(timeout)`: Same as `close` difference is that the client is NOT disposed after calling flush
+
+- `Client::dtor`: Must never block. Might throw away pending events.
```

Comment:
Do we need to specify this (particularly the name)? `__del__` in Python is just the default impl for objects.